server/config/config.go

```diff
@@ -82,8 +83,8 @@ type Config struct {
 	LogFileDeprecated  string `toml:"log-file" json:"log-file,omitempty"`
 	LogLevelDeprecated string `toml:"log-level" json:"log-level,omitempty"`
 
-	// TsoSaveInterval is the interval to save timestamp.
-	TsoSaveInterval typeutil.Duration `toml:"tso-save-interval" json:"tso-save-interval"`
+	// Tso service related configuration.
+	Tso TsoConfig `toml:"tso" json:"tso"`
```

Comment:
We need to notify other components such as operator for this change

Reply:
Could you be more detailed? The components I think are related to PD's config includes `tidb-operator` and `tiup`. Any else?